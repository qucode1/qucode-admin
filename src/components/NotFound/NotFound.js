import React from 'react'

const NotFound = (props) => (
  <div>
    <h2>Nothing to see here :(</h2>
    <p>{props.message}</p>
  </div>
)

export default NotFound
