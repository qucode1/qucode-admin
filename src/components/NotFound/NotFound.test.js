import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import NotFound from './NotFound'

describe('<NotFound />', () => {
  it('should render h2 and p with message as expected', () => {
    const wrapper = shallow(<NotFound message='test message' />)
    const tree = toJson(wrapper)
    expect(tree).toMatchSnapshot()
  })
})
