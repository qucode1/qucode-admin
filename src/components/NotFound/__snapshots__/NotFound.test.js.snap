// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render h2 and p with message as expected 1`] = `
<div>
  <h2>
    Nothing to see here :(
  </h2>
  <p>
    test message
  </p>
</div>
`;
